### ISSUE TITLE
medium logo and link to it

### ISSUE THREAD

**Issue by deniseOme** [opentargets/webapp#232]
If we are moving to Medium  now (https://medium.com/opentargets) shall we point users to it from our homepage? Similar to what UCL does?

Screen Shot 2018-01-09 at 19.48.01

What do you think @elipapa @andrewhercules?

**Comment by elipapa** [comment]
I think it's a good idea to have a medium icon amongst together with the facebook/github/linked in ones
Here is font awesome version http://fontawesome.io/icon/medium/ 
@LucaFumis is it easy to do?

We hadn't made the decision to phase out blog.opentargets.org in favour of medium.com/opentargets yet. When we last discussed it, we thought it would be a good idea to have them running for a few months in parallel.
It has no impact on infrastructure, although it can be awkward to have to publish/edit twice. 

Should we set a deadline to decide on one vs the other?

**Comment by iandunham** [comment]
You just need to decide on one, and make sure all the content is available - I think the latest release blog is not on medium. 
One minor thing to sort out with the content on medium is that it is not sorted chronologically at the moment possibly due to the process of transfer - can that be fixed?

e.g.

**Comment by deniseOme** [comment]
Now it is.

The stats didn't get formatted as in Ghost though.
Perhaps some of the markdown is lost upon importing from Ghost to Medium?!?!?!?

<img width="723" alt="screen shot 2018-01-11 at 13 41 30" src="https://user-images.githubusercontent.com/6472381/34828379-4e4fab7e-f6d5-11e7-95bf-8c6fea256dfc.png">

**Comment by LucaFumis** [comment]
I'll add this (and youtube) icon to the footer; the side tab might get a little busy - I'll need to try.
Pity the icon in fontawesome is still the old logo - I had a look at it back in nov/dec and was hoping they would have updated it by now...

**Comment by emepyc** [comment]
I don't think they will update given that fontawesome 5 is out now...
https://fontawesome.com/?utm_source=font_awesome_homepage&utm_medium=display&utm_campaign=fa5_released&utm_content=banner

M;

**Comment by deniseOme** [comment]
If it gets too busy, we can remove some icons (either the ones that are doing well e.g. twitter, linkedin) or we can drop facebook.

**Comment by elipapa** [comment]
we should measure how many clicks we get from the sidebar and the footer. can’t we add an utm code?

________________________________

**Comment by LucaFumis** [comment]
I've added both Medium and YouTube logos (as per issue #231) here:
https://new-social-media--paratrooper-antelope-70721.netlify.com/

Slightly increased size in the footer, slightly more compact in the side tab.

I looked into upgrading to FontAwesome 5, which features the latest Medium logo.
The migration is not so straightforward (syntax changed, automatic scripts didn't work and the FontAwesome shim file approach broke random things throughout the app) so we decided to leave it until we revise the tech stack and for the app.

If we're happy I'll merge the changes.

**Comment by peatroot** [comment]
Looks good to me @LucaFumis.

**Comment by andrewhercules** [comment]
Looks good @LucaFumis. 

And I agree 100% - we can make a larger project around upgrading to FA5 as we know that the shim file breaks various aspects of the Platform (e.g. the icon beside the profile link on the associations page is resized).

**Comment by LucaFumis** [comment]
Merged into `master` and production `v3.4`
